controllers/goroutines/bedrock_operators.go

```diff
@@ -0,0 +1,116 @@
+//
```

Comment:
How about changing the name of this file to `operator_status`?

Reply:
sure, good idea